_data/projects/internet-speed-testing.yml

```
@@ -0,0 +1,20 @@
+---
+name: Internet speed testing
+desc: >-
+  Android library for test internet speed implement by kotlin language!
+site: https://github.com/oatrice/internet-speed-testing
+tags:
+- android
+- libraries
+- kotlin
+- internet
+- speed
+- testing
+- measurement
+- hacktoberfest
+upforgrabs:
+  name: help wanted
+  link: https://github.com/oatrice/internet-speed-testing
```

This link should go directly to the list of issues available for new contributors:

```suggestion
  link: https://github.com/oatrice/internet-speed-testing/labels/help%20wanted
```